site/joseki.js

```diff
@@ -825,6 +830,12 @@ const EMPTY_SCORE = {
         }
     }
 
+    function emptyStartPoint() {
+        shutdown();
+        boardMsg("No josekis yet!", 'black');
```

Comment:
"Joseki" is the plural except in variable names where I do "joseki" vs "josekis" for single vs collections.